contracts/strategies/BenqiStrategyForLP.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "../YakStrategy.sol";
+import "../interfaces/IBenqiStakingContract.sol";
+import "../interfaces/IPair.sol";
+import "../lib/DexLibrary.sol";
+
+/**
+ * @notice Benqi strategy for staking QI-AVAX.
+ */
+contract BenqiStrategyForLP is YakStrategy {
+    using SafeMath for uint;
+
+    IBenqiStakingContract private stakingContract;
+    IERC20 private wavaxRewardToken;
+    IERC20 private qiRewardToken;
+    IPair private swapPairToken; // WAVAX-QI LP
+
+    constructor (
+        string memory _name,
+        address _depositToken,
+        address _wavaxRewardToken,
+        address _qiRewardToken,
+        address _stakingContract,
+        address _swapPairToken,
+        address _timelock,
+        uint _minTokensToReinvest,
+        uint _adminFeeBips,
+        uint _devFeeBips,
+        uint _reinvestRewardBips
+    ) {
+        name = _name;
+        depositToken = IERC20(_depositToken);
+        wavaxRewardToken = IERC20(_wavaxRewardToken);
```

Comment:
rewardToken which is defined in YakStrategy also needs to be assigned here, it is the token used by the front-end and botters to figure out what the reward they'll be receiving, and to report in the front-end on the reinvest button